src/client/Microsoft.Identity.Client/Platforms/iOS/Broker/BrokerKeyHelper.cs

```diff
@@ -141,24 +141,21 @@ internal static string DecryptBrokerResponse(string encryptedBrokerResponse, ILo
                 MsalErrorMessage.iOSBrokerKeyFetchFailed);
         }
 
-       private static AesManaged CreateSymmetricAlgorithm(byte[] key)
+       private static Aes CreateSymmetricAlgorithm(byte[] key)
```

Comment:
Why were these changes made? Wondering if there are any hidden behavior changes because of this.

Reply:
This change was needed because `AesManaged` is showing up as deprecated in net6, and we'd rather not have deprecated crypto APIs. 

Communication with the broker would not be possible without this crypto operation, so functionally it works.